setup.cfg

```
@@ -5,7 +5,7 @@ url = https://github.com/aimakerspace/PeekingDuck
 author = 'AI Singapore'
 author_email = cvhub-ext@aisingapore.org
 keywords = 'peekingduck', 'computer vision', 'inference', 'edge'
-description = A simple tool for Computer Vision Inference
+description = A modular framework built to simplify Computer Vision Inference Workloads.
```

This will be our tagline so we need to put some thought into it. If you're okay with this suggestion I will change the Github description as well.
```suggestion
description = A modular framework that simplifies computer vision inference.
```

---

i prefer "workloads" because its more than just CV model inference. we do preprocessing, post processing, and different forms of heuristics/calculations too. Perhaps find a better word to replace "workloads"?

---

Good point - let's go with your original proposal without the caps.
```suggestion
description = A modular framework built to simplify Computer Vision inference workloads.
```